setup.py

```diff
@@ -1,13 +1,17 @@
+from __future__ import absolute_import
```

Comment:
@feanil facing issues with git commit.

```
    error in django-oauth-plus setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Invalid requirement, parse error at "'+https:/'"
```
Tried different things also in setup.py but each time it fails.

Reply:
you can remove this.